src/main/resources/features/theme 1/25 - site review.feature

```diff
@@ -0,0 +1,60 @@
+@regression
+  Feature: 25 - site review
+
+    Scenario: VE user creates a satisfactory site review with an activity performed
+      #Search for a site with an existing site review
+      Given I load "SITE_REVIEW" as {siteNumber}, {siteName}
+      And I login without 2FA using "VEHICLE_EXAMINER_USER" as {vehicleExaminer}
+      And I search for Site information by site number with {siteNumber}
+      #Start a site review
+      When I click the "Site review" link
+      And I click the "Record a site review" link
+      And I enter the date of site visit as 1 days ago
+      And I enter "John Doe" in the field with id "aeName"
+      And I enter "AE" in the field with id "aeRole"
+      And I press the "Save and continue" button
+      #And I record a "satisfactory" compliance outcome
+      And I click the button with id "change_CO_button"
+      And I click the "Satisfactory" radio button
+      And I press the "Save and return" button
+      #Record a satisfactory premises outcome
```

Comment:
It would be nice to split these steps up into what is doing on each page, making it easy to see what steps are carried out on what pages